PerimeterXModule/DataContracts/Requests/Additional.cs

```diff
@@ -28,5 +28,18 @@ public class Additional
 
 		[DataMember(Name = "px_cookie_hmac")]
 		public string PxCookieHMAC;
+
+		[DataMember(Name = "cookie_origin")]
+		public CookieOrigin CookieOrigin;
+
+		[DataMember(Name = "original_uuid")]
+		public string OriginalUUID;
+
+		[DataMember(Name = "original_token_error")]
+		public string OriginalTokenError;
+
+		[DataMember(Name = "decoded_original_token")]
+		public object DecodedOriginalToken;
```

Comment:
Don't know how consistent you want to be, on node it is sent as `px_decoded_original_token` ([here](https://github.com/PerimeterX/perimeterx-node-core/blob/b9b1569697d9de6c42388443589be1b847198903/lib/pxapi.js#L75)).